libbeat/docs/processors-using.asciidoc

```diff
@@ -1184,6 +1184,28 @@ The following settings are supported:
                   empty array (`[]`) or an empty object (`{}`) are considered
                   empty values. Default is `false`.
 
+[[fingerprint]]
+=== Generate a fingerprint of an event
+
+The `fingerprint` processor generates a fingerprint of an event based on a
+specified subset of its fields.
+
+[source,yaml]
+-----------------------------------------------------
+processors:
+ - fingerprint:
+     fields: ["field1", "field2", ...]
+-----------------------------------------------------
+
+.Fingerprint options
+[options="header"]
+|======
+| `fields`          | yes       |               | List of fields to use as the source for the fingerprint.                                                               |
+| `target_field`    | no        | `fingerprint` | Field in which the generated fingerprint should be stored.                                                             |
```

Comment:
Tables with more than 3 columns don't look very good in our HTML output (see screen capture in previous comment). Docbook swallows all table attributes, so we can't do anything about that right now. You could remove the example row and provide the example as part of the description. Or wait for some shift in the universe that will make this right.

Reply:
I replaced the table with a definition list, like the one used in the Extract Array processor, for instance. Let me know what you think. Thanks!